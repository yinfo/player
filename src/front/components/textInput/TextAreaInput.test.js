// @flow

import React from 'react';
import { shallow } from 'enzyme';
import TextAreaInput from './TextAreaInput';

describe('TextAreaInput component', () => {
  it('renders as expected', () => {
    const props = {
      label: '',
      id: '1',
      nbrows: 1,
      value: '',
      onChange: jest.fn(),
    };
    const component = shallow(<TextAreaInput {...props} />);
    expect(component).toMatchSnapshot();
  });
});
