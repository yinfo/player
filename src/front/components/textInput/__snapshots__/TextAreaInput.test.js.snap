// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextAreaInput component renders as expected 1`] = `ShallowWrapper {}`;
